real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GpsPanel.java

```diff
@@ -95,7 +96,7 @@ public void actionPerformed(String topicName, Message m) {
 		});
 
 		this.add(map);
-		map.repaint();
+//		map.repaint();
```

Comment:
because if we leave the map up, the UI thread will get completely flooded. This is also the same for the camera view